### PR TITLE
Update james-project SHA-1

### PR DESCRIPTION
**Pull request by Arsnael**
Let's let the CI check that before we merge now :)

### CODE REVIEW

File: openpaas-james/apps/distributed-ldap/src/main/java/com/linagora/openpaas/james/app/DistributedLdapServer.java [hunk 1]
```diff
@@ -58,7 +60,7 @@
                 new RabbitMailQueueRoutesModule(),
                 new RabbitMQEventBusModule(),
                 new DistributedTaskSerializationModule()))
-        .with(new LdapUsersRepositoryModule(), new CustomMethodModule());
```
Comment (Arsnael) [inline openpaas-james/apps/distributed-ldap/src/main/java/com/linagora/openpaas/james/app/DistributedLdapServer.java:61]: My guess is here we should even probably delete the distributedLdapServer, and let the LDAPModuleChooser do its job? If yes, probably other PR? And how should we organize that?

Comment (chibenwa) [reply openpaas-james/apps/distributed-ldap/src/main/java/com/linagora/openpaas/james/app/DistributedLdapServer.java:61]: > My guess is here we should even probably delete the distributedLdapServer, and let the LDAPModuleChooser do its job?

Yes

> If yes, probably other PR?

Ok for me.

> And how should we organize that?

Just drop the distributedLdap server main, keep the test as is with the distributed server configured. For the deployment test we likely should use a DockerFile to customize the usersrepository.xml file.

I'm in favor of dropping the JMAP integration tests for LDAP.

**Review by chibenwa — APPROVED** [review]

**Comment by Arsnael** [comment]
```
[ERROR] Failures: 
[ERROR]   DistributedLdapLinagoraEchoMethodTest.echoMethodShouldRespondOKWithRFC8621VersionAndSupportedMethod 1 expectation failed.
Expected status code <200> but was <401>.

[ERROR]   DistributedLdapLinagoraEchoMethodTest.echoMethodShouldRespondWithRFC8621VersionAndUnsupportedMethod 1 expectation failed.
Expected status code <200> but was <401>.

[ERROR]   DistributedLdapLinagoraEchoMethodTest.echoMethodShouldReturnUnknownMethodWhenMissingCoreCapability 1 expectation failed.
Expected status code <200> but was <401>.

[ERROR]   DistributedLdapLinagoraEchoMethodTest.echoMethodShouldReturnUnknownMethodWhenMissingLinagoraEchoCapability 1 expectation failed.
Expected status code <200> but was <401>.
```
I can't seem to reproduce that locally (green for me) so I'm a bit confused TBH...

File: openpaas-james/apps/distributed-ldap/src/main/java/com/linagora/openpaas/james/app/DistributedLdapServer.java (hunk 1 above)
Comment (Arsnael) [reply openpaas-james/apps/distributed-ldap/src/main/java/com/linagora/openpaas/james/app/DistributedLdapServer.java:61]: https://github.com/linagora/openpaas-james/issues/56

**Review by chibenwa — APPROVED** [review]

**Pull request merged**